src/tasks-check.js

```diff
@@ -0,0 +1,25 @@
+export function checking(array) {
+  array.forEach((tsk) => {
+    const checkbox = document.getElementById(`${tsk.index}-checkbox`);
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        tsk.completed = true;
+        window.localStorage.setItem('to-do-list', JSON.stringify(array));
+      } else {
+        tsk.completed = false;
+        window.localStorage.setItem('to-do-list', JSON.stringify(array));
+      }
+    });
+  });
+}
```

Comment:
The following is just a recommendation on improvement:
- [ OPTIONAL ] It would be better to extract the code that updates the localStorage into a separate function, instead of repeating it twice in the 'if' and 'else' branches of the event listener.

Reply:
Hmm... Great I'll use the recommended approach. Thanks for your construtive suggestion.